src/Native/Runtime/GCHelpers.cpp

```diff
@@ -220,3 +220,63 @@ COOP_PINVOKE_HELPER(void, RhGetMemoryInfo, (UInt32* highMemLoadThreshold, UInt64
                                                        lastRecordedMemLoad,
                                                        lastRecordedHeapSize, lastRecordedFragmentation);
 }
+
+EXTERN_C REDHAWK_API Int64 __cdecl RhGetTotalAllocatedBytes(Boolean precise)
+{
+    if (!precise)
+    {
+#if defined(_TARGET_AMD64_) || defined(_TARGET_ARM64_)
```

Comment:
In CoreCLR this is using `_TARGET_64BIT_`, this symbol appears undefined for corert so i've used these two

Reply:
You can just use `#ifdef BIT64` here